test/test_quantized.py

```diff
@@ -267,17 +268,49 @@ def test_qadd_relu_different_qparams(self):
         self.assertEqual(qCrelu_hat, qCrelu_out_hat,
                          message="AddReLU.out failed")
 
+    """Tests the correctness of the scalar addition."""
+    @given(A=hu.tensor(shapes=hu.array_shapes(1, 4, 1, 5),
+                       elements=st.floats(-1e3, 1e3, allow_nan=False),
+                       qparams=hu.qparams()),
+           # b < 0 is WIP.
+           b=st.floats(min_value=0, max_value=1e3, allow_nan=False))
+    def test_qmul_scalar_relu(self, A, b):
+        mul_scalar = torch.ops.quantized.mul_scalar
+        mul_scalar_relu = torch.ops.quantized.mul_scalar_relu
+
+        A, (scale, zero_point, dtype) = A
+        if dtype == torch.qint32:
+            return
+
+        A = A.astype(np.float32)
+        qA = torch.quantize_linear(torch.from_numpy(A), scale, zero_point, dtype)
+
+        C_ref = qA.dequantize() * b
+        C_hat = mul_scalar(qA, b)
+
+        C_relu_ref = qA.dequantize() * b
+        C_relu_ref = torch.relu(C_relu_ref)
+        C_relu_hat = mul_scalar_relu(qA, b)
+
+        self.assertEqual(C_ref, C_hat.dequantize(), prec=1,
```

Comment:
prec seems too loose. Do we need 1?
Can we match the reference implementation to get a better match?

Reply:
I think at this point this is OK. Once we have a more rigorous guidelines for reference implementation and once we have a clear understanding how to mimic the double rounding, we can change that. Right now, it works with `prec=0.1`, but I think this is data specific